fastembed/sparse/splade_pp.py

```
@@ -135,6 +137,21 @@ def load_onnx_model(self) -> None:
             device_id=self.device_id,
         )
 
+    def tokenize(self, texts: Union[str, Iterable[str]], **kwargs: Any) -> list[Encoding]:
```

Ruff (0.14.1)

Unused method argument: `kwargs`

(ARG002)